src/main/java/org/opensearch/performanceanalyzer/collectors/telemetry/RTFDisksCollector.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.performanceanalyzer.collectors.telemetry;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.opensearch.performanceanalyzer.OpenSearchResources;
+import org.opensearch.performanceanalyzer.commons.OSMetricsGeneratorFactory;
+import org.opensearch.performanceanalyzer.commons.collectors.DisksCollector;
+import org.opensearch.performanceanalyzer.commons.collectors.PerformanceAnalyzerMetricsCollector;
+import org.opensearch.performanceanalyzer.commons.metrics.AllMetrics;
+import org.opensearch.performanceanalyzer.commons.metrics.MetricsConfiguration;
+import org.opensearch.performanceanalyzer.commons.metrics_generator.DiskMetricsGenerator;
+import org.opensearch.performanceanalyzer.commons.metrics_generator.OSMetricsGenerator;
+import org.opensearch.performanceanalyzer.commons.stats.metrics.StatExceptionCode;
+import org.opensearch.performanceanalyzer.commons.stats.metrics.StatMetrics;
+import org.opensearch.telemetry.metrics.Histogram;
+import org.opensearch.telemetry.metrics.MetricsRegistry;
+import org.opensearch.telemetry.metrics.tags.Tags;
+
+public class RTFDisksCollector extends PerformanceAnalyzerMetricsCollector {
+
+    private Histogram DiskWaitTimeMetrics;
+    private Histogram DiskServiceRateMetrics;
+    private Histogram DiskUtilizationMetrics;
+
+    private MetricsRegistry metricsRegistry;
+    private static final Logger LOG = LogManager.getLogger(RTFDisksCollector.class);
+
+    public RTFDisksCollector() {
+        super(
+                MetricsConfiguration.CONFIG_MAP.get(DisksCollector.class).samplingInterval,
```

Comment:
Why are we piggybacking on DisksCollector.class setting?

Reply:
To match the collector periodicity. If needed, we can create new items in CONFIG_MAP for new collectors.

Reply:
Let's create new items please.